\chapter{Genome-wide survey of misexpressed genes in F2 hybrid yeast}
\label{chpt:misexpression}

\section{Introduction}

%$<$F1 hybrids frequently used in agriculture/artificial breeding to combine beneficial traits of 2 genetic backgrounds (example of beneficial trait: hybrid vigor)$>$

F1 hybrids are commonly used in commercial breeding to combine beneficial traits of two inbred parental lines. A common beneficial trait in F1 hybrids is an increased growth rate, termed ``hybrid vigor''. Hybrid vigor is observed in many different clades---from mules \cite{Hutt1952}, to lentils \cite{Tan2022}, to brewing yeasts \cite{Gibson2017}. It is not entirely clear why hybrid vigor is such a commonly observed trait of so many different hybrids. One hypothesis is that the high degree of heterozygosity increases the chance of hybrids harboring beneficial single variants, and that different beneficial variants are significant for different aspects of the organismal phenotype, or different environments the organism might encounter \cite{Collins1921, Shapira2016, Zhang2020}. Alternatively, or additionally, epistatic combinations of hybrid loci can create new effects not seen in either parent \cite{Naseeb2021}.

%$<$High heterozygosity in hybrids is thought to contribute to their beneficial traits. But maintaining each allele in its native genomic context is undoubtedly important too b/c F1s are the best and subsequent generations get worse (F3, F3, etc.)$>$

Hybrid vigor is most strongly exhibited in F1 hybrids. Subsequent generations (F2, F3, etc.) often show substantially worse growth phenotypes than the F1 despite still harboring a significant amount of heterozygosity \cite{Larson1944}. This rapid decline in vigor over a small number of generations suggests that the relationship between heterozygosity and vigor is not additive \cite{Hallahan2018}. Heterozygosity instead appears to promote vigor only when specific combinations of parental alleles are present \cite{Wang2015}.

% I think this is cool for the project, but not relevant for the chapter right now
%What aspects of the genomes of F2 hybrids and subsequent generations might be detrimental to vigor? One possibility is recombination: genes outside of their native chromosomal context may be more liable to be improperly expressed or not expressed at all. Another possibility is changes in the dosage of parental alleles, as the F2 generation and beyond typically can no longer guarantee the even balance of both parents' alleles that the F1 exhibits. While these two factors are typically confounded, yeast genetics allows for them to be studied independently. Yeast meiosis produces four recombined haploid gametes from every somatic precursor cell. All four of these gametes, termed ``spores'' are potentially viable to begin a new haploid clonal population. The combination of all four F2 spores produced from one F1 hybrid precursor cell still contains an even balance of both parent's alleles, but now in a recombined genomic context. Often, pairs of these spores can be crossed to produce diploid F2 hybrid yeasts that preserve an equal balance of parental alleles, one copy of each gene. These F2s therefore only differ from the F1s by the presence of recombination.

%$<$What combinations are important? Are there any patterns?$>$

What combinations of parental alleles could be responsible for hybrid vigor? To bestow hybrids with novel or enhanced traits, these combinations of alleles would need to have new effects in the hybrid, not observed when either parental allele is in its own parental background. Strategies to scan genomes for phenotypic effects resulting specifically from combinations of parental alleles are beginning to become feasible. These strategies require significant genetic manipulation to hold certain portions of the genome fixed while testing variation at other loci \cite{Buzby2025}. Another strategy that can be used to lessen the combinatorial search space is to begin scanning for epistatic effects at the level of transcription rather than phenotype.

%$<$Gene expression measurements can serve as sensors of the molecular environment.$>$

Gene expression measurements can serve as sensors of the molecular environment. If factors influencing a genes' expression are additive, the hybrid will have a total expression level that will fall within the range of both parents' expression. If new interactions are present in the hybrid that affect the gene, its expression will be outside of the range of the parental expression and we say it is misexpressed \cite{Landry2005}. Characterizing the degree of misexpression in F1 hybrids serves as a measure of the total number of possible pairwise epistatic interactions that can affect gene expression. The extent to which these misexpression events are recorded in F2 progeny can subsequently be examined to begin to associate certain combinations of parental loci with specific misexpressed genes. This method does not detect all possible epistatic interactions but serves as an unbiased survey of one type of epistasis: epistasis that affects gene expression.

%$<$Present study$>$

Here we survey the extent of misexpression in F1 hybrid yeast from a cross between \textit{Saccharomyces cerevisiae} and \textit{Saccharomyces paradoxus}. We detect 103 genes that were underexpressed and 33 genes that were overexpressed consistently in the F1 across 6 different batches of standard laboratory conditions. We next began to survey whether any of these genes were also misexpressed in F2 hybrids harboring different combinations of parental alleles. We collected 22 F2 hybrid samples by inducing sporulation in a newly developed F1 hybrid system which disables a meiosis checkpoint that typically acts as a reproductive barrier between these species \cite{Bozdag2021}. We found that transcriptomic data alone was sufficient to map recombination breakpoints in these F2 hybrids. We lastly assessed the expression of misexpressed F1 genes in our 22 F2 hybrid samples. We found a systematic bias of these misexpressed genes to be misexpressed in the opposite direction from what was observed in F1s. This reversed misexpression could be explained for the set that were overexpressed in the F1 and underexpressed in the F2 based on their above-average expression level in the F1s, as genes that were not misexpressed but had comparable expression levels showed the same pattern. The set that were underexpressed in the F1s and overexpressed in the F2s, however, could not be explained by expression level alone. Taken together, these results demonstrate a method for assessing misexpression in F1 and F2 hybrid yeast and suggest that properties of hybrid genomes that are not specific to certain combinations of parental alleles are possibly also influencing the underexpression of F1 hybrid genes.

\section{Results \& Discussion}

\subsection{Identifying misexpressed genes F1 hybrid}

To characterized the number of misexpressed genes in the F1 hybrid, we defined a parental expression range for each gene and tested whether or not the expression of the F1 hybrid fell within this range. We used transcriptomic data from the six environments studied in Chapter \ref{chpt:plasticity} but restricted to samples from the starting timepoint, which were taken in standard laboratory conditions. These samples allowed us to assess the prevalence of misexpression in standard laboratory conditions that was robust to batch effects, such as which laboratory and year the data were collected in. These data consisted of 86 parental RNAseq samples and 45 F1 hybrid RNAseq samples sampled in 6 different batches of standard laboratory environments.

We defined a gene as misexpressed if the middle 50\% of F1 hybrid counts, average of both hybrid alleles, were outside the range defined by the median expression of both parents (Fig \ref{fig:id}A). We chose this definition because it most closely aligned with definitions in prior papers \cite{Landry2005, McManus2014} while still allowing for some counts in F1 hybrids to overlap the range of parental counts. We reasoned this overlap was appropriate because the diversity of environments the expression data was collected in created higher variability between samples of the same gene. Using this definition, we found that 3\% of genes were misexpressed, with 33 genes being overexpressed and 103 genes being underexpressed in the F1 hybrid (Fig \ref{fig:id}B).

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Misexpression/Fig_ID.png}}
    \caption{\textbf{Identifying misexpressed genes in the F1 hybrid.} A) Illustration of misexpression test. For a single gene, the range of parental expression is defined as the range between the \textit{S. cerevisiae} and \textit{S. paradoxus} median counts. A gene is considered misexpressed in the F1 hybrid if the middle 50\% of its counts (between the 25\% and 75\% quantile) does not overlap this range. B) Counts of not misexpressed, overexpressed, and underexpressed genes in F1 hybrids. C) Allele-specific misexpression when the misexpression test is repeated on the two sets defined in B using only counts from one F1 hybrid allele, \textit{S. cerevisiae} (Hyc) or \textit{S. paradoxus} (Hyp).}
    \label{fig:id}
\end{figure}

To identify our sets of over- and underexpressed genes, we only considered the F1 hybrid expression as the average expression of both hybrid alleles. To assess how often one parental allele was driving this misexpression, we repeated the same misexpression test separately for each allele in the sets of 33 overexpressed and 103 underexpressed genes. We found that about a third of both overexpressed and underexpressed genes had misexpression driven by aberant expression of both alleles (Fig \ref{fig:id}C). The remaining two-thirds of misexpressed genes were driven by a single allele. In both overexpressed and underexpressed genes, that allele was slightly more likely to be the \textit{S. cerevisiae} allele.

\subsection{The extent of recombination is readily assessed in F2 haploid spores using transcriptomes}

Our ultimate objective was to determine how often genes that were identified as misexpressed in the F1 hybrid remained misexpressed in a recombined hybrid genome. To begin to address this, we collected transcriptiome data from F2 hybrids. We induced sporulation in our F1 hybrid yeast that contain a disabled meiosis checkpoint to allow for recombination between parental genomes (see Methods, and \cite{Bozdag2021}). We then dissected over 200 F2 hybrid spores. A pilot sample of 24 of these haploid spores was submitted for 3' mRNA tagmentation sequencing. 22 of these samples had sufficient growth to allow for sequencing. 

To determine whether recombination occurred in our F2 spores, we organized our expression counts in genome order and asked which hybrid allele had more counts for each gene. We reasoned that if recombination occurred, we would see genes switch from having higher counts of one parental allele to the other at some point along the chromosome. We indeed found that contiguous genes tended to have higher counts of the same parental allele (Fig \ref{fig:map}A). On average one to two points in each arm of each chromosome, these counts transitioned to be higher for the other parental allele, indicating that recombination in our F2 spores was  detectable using transcriptomic data alone.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Misexpression/Fig_Map.png}}
    \caption{\textbf{Assessing recombination in F2 haploid spores} A) Alleleic expression from a randomly-selected sample. Each point is a gene, colored by whether the Scer or Spar allele of the gene had more reads mapped to it. Each arm of each of the 16 chromosomes is plotted separately with genes arranged in the order they appear along the chromosome arm (distances are not proportional to base pair distances). Maps of the other 21 samples are presented in Appendix \ref{chpt:ChromMaps}.}
    \label{fig:map}
\end{figure}

\subsection{Genes identified as misexpressed in the F1 hybrid tend to be misexpressed in F2 haploid spores in the opposite direction}

To assess how often genes remained misexpressed in our F2 haploid spores, we first asked how similar the expression of most genes was between F2 haploid spores and the F1 diploid. As different haploid spores had different alleles for each gene, we quantified expression independently for each gene by taking its median expression separately for each allele. For each allele of each gene, we selected the subset of haploid samples that had higher expression of that allele and calculated the median expression among them. We then repeated for the other allele by calculating the median expression among the remaining samples. We found that 12 genes had no samples with higher read counts of the \textit{S. cerevisiae} allele and 14 genes had no samples with higher read counts of the \textit{S. paradoxus} allele, so these 28 genes were excluded.

We found that the median expression among the remaining genes in the F1 diploid was weakly correlated with the median expression in the F2 haploid ($R^2$ = 0.17), for both the \textit{S. cerevisiae} and \textit{S. paradoxus} alleles. A similar $R^2$ value was obtained when correlating the F1 diploid expression of the \textit{S. paradoxus} allele with the F2 haploid expression of the \textit{S. cerevisiae} allele and vice-versa ($R^2$ = 0.16), suggesting that \textit{cis}-regulatory variation between alleles is a comparatively small source of this transcription divergence compared to global factors influencing the transcriptome. These global factors include the fact that the F1 diploid was made from different parental strains than the F2 (\textit{S. cerevisiae} strain S288c and \textit{S. paradoxus} strain CBS432 in the F1, and \textit{S. cerevisiae} strain W303 and \textit{S. paradoxus} strain N17 in the F2), transcriptomes were collected in separate labs and in separate years, and that diploid yeasts tend to be less stressed than haploid yeasts \hl{cite?}.

For all these reasons, we were surprised to see that the 136 genes identified as misexpressed in the F1 hybrid displayed much more predictable expression variation in the F2 hybrid. Specifically, median expression of F2 misexpressed genes tended to be misexpressed in the opposite direction from what was observed in the F1 hybrid. While this was true for both the undexpressed F1 genes (Fig \ref{fig:capt}A and E) and the overexpressed F1 genes (Fig \ref{fig:capt}C and G) of both parental alleles, it was strikingly clear among the 33 genes identified as overexpressed in the F1 hybrid.

We reasoned that genes that were overexpressed or underexpressed in the F1 hybrid might tend to be on the extreme tails of the distribution of possible gene expression values. Then, simply by chance, those misexpressed genes may tend to have less-extreme expression in the F2 hybrid. This would create the pattern we observe of F2 genes being misexpressed in the opposite direction from F1 genes, as they would be trending towards the middle of the distribution of possible gene expression values. To test this possibility, we matched each misexpressed F1 gene with a corresponding non-misexpressed F1 hybrid gene of similar median expression (within 5\% of the misexpressed gene). We found that this extreme tails hypothesis was sufficient to explain the F2 expression of genes that were overexpressed (Fig \ref{fig:capt}, compare C vs. D and G vs. H) but not underexpressed (Fig \ref{fig:capt}, compare A vs. B and E vs. F) in the F1 hybrid. Among the set of non-misexpressed genes that were matched to the underexpressed set, F2 median expression was equally likely to be above or below the F1 median expression.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Misexpression/Fig_Hyc.png}}
    \label{fig:hyc}
\end{figure}

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Misexpression/Fig_Hyp.png}}
    \label{fig:hyp}
\end{figure}

\begin{figure}
    \centering
    \caption{\textbf{Expression of overexpressed and underexpressed F1 genes and control gene sets.}  A-D) \textit{S. cerevisiae} allele. E-H) \textit{S. paradoxus} allele. A) The set of genes that were underexpressed in the F1. B) A set of non-misexpressed genes of comparable expression level to the set in A. C) The set of genes that were overexpressed in the F1. D) A set of non-misexpressed genes of comparable expression level to the set in C. E-H) same as A-D) for \textit{S. paradoxus} allele.}
    \label{fig:capt}
\end{figure}

\section{Conclusions \& Future Directions}

Here we begin to investigate how recombination of parental genomes affects the set of genes that are misexpressed in the F1 hybrid. We have identified as set of 136 genes that were overexpressed or underexpressed in F1 hybrids in standard laboratory conditions. We have additionally produced F2 haploid spores and determined, using transcriptomic data, that recombination between parental genomes has occurred, averaging 1-2 breakpoints per chromosome arm. We have lastly begun to assess the extent of misexpression in the F2 haploid spores, using a pilot set of 22 transcriptomes. This assessment has found that the sets of genes that were overexpressed and underexpressed in the F1 hybrid have different properties. Overexpressed genes tend to be the most highly expressed genes in the F1 hybrid. These same genes do not tend to be as highly expressed in the F2 hybrid, resulting in a systematic downgrade in their expression in the F2. Genes that are underexpressed in the F1 hybrid, however, do not tend to have extreme expression. In this underexpressed set, F2 expression tends to be higher than F1 expression for the same gene. This is more likely to be true for misexpressed genes than a control set of non-misexpressed genes with the same median expression levels. 

It is presently unclear why these genes that were underexpressed in the F1 hybrid tend to be more highly expressed in the F2 hybrid. As this tendency was seen systematically across all 22 F2 hybrids, it is unlikely to be caused by specific combinations of parental alleles. One possibility is that the cause of the F1 underexpression may be genetic interactions between the two alleles of the underexpressed gene itself, or between alleles of a regulator of the gene. Our F2 haploids, possessing only one allele of each gene, would not exhibit these interactions. This possibility can be tested by crossing our F2 haploids to obtain diploids and repeating our transcriptome analysis.

The extensive variation between the F1 hybrid and F2 hybrid expression can likely be reduced significantly by increasing the number of F2 hybrid samples and collecting additional F1 samples derived from the same parental strains as F2. At present, there are on average 11 F2 samples contributing to expression estimation of each gene while 45 F1 hybrid samples were used in this study. As a result, F2 estimations are more subject to noise among replicates, which may contribute to the lack of correlation between F1 and F2 gene expression. Introducing multiple genetic backgrounds by using data from different parental strains further contributes to the lack of expression correlation and should therefore be standardized in the full study.

We have demonstrated the feasibility of measuring F1 misexpressed genes in an F2 recombined genome. We have additionally detected a potential systematic difference between over and underexpressed genes based on their expression in F2 hybrids. Collecting a full dataset of F2 haploid and diploid transcriptomes will grant us the statistical power to associate certain combinations of parental alleles with the misexpression of specific genes.