\chapter{Comparison of signal propagation through the low nitrogen response network in two yeast species}
\label{chpt:networks}

\section{Introduction}
%\label{sec:intro}

Signal propagation through gene regulatory networks is a vital process through which organisms respond to changes in their environment. Signaling begins when environment-sensing proteins, alter their structure or function due to a change in cellular state, triggering a gene regulatory cascade \cite{Bahn2007}. This cascade ultimately results in the upregulation of genes whose products play important roles in adjusting to the environmental change. For example, upregulation of enzymes involved in breaking down alternative carbon sources in response to a glucose shortage \cite{Gancedo1998}. Recent studies have found surprising variation in how environmental signals transduce through signaling networks, even between closely related species \cite{Britton2020}. This variation has important implications for how regulatory networks evolve. In particular, how much neutral space---variation of little phenotypic consequence---is available in signaling networks structure for populations to explore over evolutionary time \cite{Payne2015}.

The intermediaries in a signal transduction response are transcription factors (TFs). A common strategy to assess how genes are connected in a regulatory network is to examine which genes have altered mRNA abundance in response to repression of specific TFs. Recent studies in yeast have taken advantage of gene editing tools and a relatively small set of TFs (169 recorded in the Yeastract database \cite{Hahn2011}) to compare how the same set of TF knockouts affect gene expression in different genetic backgrounds. Surprisingly, these studies have found that the majority of differentially expressed genes in each TF knockout line are non-overlapping sets. In fact, the sets of differentially expressed genes between \textit{S. cerevisiae} and \textit{S. pombe} in response to an orthologous gene deletion have as much overlap as the gene sets from a randomly selected pair of non-orthologous gene deletions \cite{Li2025}. The majority of effects tend to be measured in genes located far from where the TF bound, suggesting these are largely pleiotropic \cite{Mahendrawada2025}. The minority of genes with conserved effects between strains tend to remain conserved between more distantly related species, but it is unclear whether these effects are more likely to have evidence of direct TF binding \cite{Liu2024}. Taken together, these studies suggest that this lack of predictability in which genes will be differentially expressed in response to a given TF perturbation is the result of widespread pleiotropic expression responses that change rapidly with evolutionary time. Within a specific genotype, however, these effects are intriguingly consistent. The identity, magnitude, and direction of differentially expressed genes are highly repeatable in multiple measurements of the same genetic background and can be detected within 30 minutes of the onset of TF repression \cite{Mahendrawada2025}.

At present, there is little mechanistic understanding of why certain genes will display a pleiotropic response to a given TF perturbation. Cells that are missing a highly connected regulator are expected to be in a stressed state. The pleiotropic effects may be a general response to this stress state. Different genetic backgrounds may exhibit different generalized stress responses, resulting in significant non-overlap between differentially expressed genes. A single, generalized stress response, however, cannot explain all the observed effects of TF knockouts. This is because the set of differentially expressed genes in response to a given TF knockout are highly dependent on the identity of the TF \cite{Liu2024}. These TF-specific effects suggest that the probability of a gene being differentially expressed upon TF perturbation is related to its position in the regulatory network relative to the perturbed TF. Determining what aspects of network position are most predictive of regulatory relationships is an open-ended question. One strategy is to evaluate the effects of TF knockouts in the context of a well studied signaling pathway, where the regulators are known and response genes can be measured. The effects of TF knockouts can be assessed by asking a) whether the deleted TF is a known component of the signaling pathway or not, and b) how the differentially expressed genes in that TF knockout are related to the set that responds during wildtype signaling. This type of analysis requires knowledge of a well characterized signaling pathway and time series expression data of a response to the induced signal.

%Connecting between the TF and a given target gene could be as simple as one degree of separation, for example via a secondary regulator that in turn directly regulates the target gene, or it might require multiple degrees of separation. Connections in the network may represent DNA binding interactions, as when a TF binds to the \textit{cis}-regulatory region of the target gene. More complicated connections are also possible, such as when a co-factor is recruited to bind the DNA or the TF indirectly influences a direct regulator of the affected gene via a change in cellular state.

%Systematically testing this possibility would require complete knowledge of every possible regulatory interaction and the environmental conditions in which that interaction can take place. In lieu of complete knowledge, any instance where there a regulatory relationship cannot be drawn between the perturbed TF and a given differentially expressed gene would either be the result of a true lack of interaction or a lack of data, and therefore inconclusive. 

The Low Nitrogen response pathway is an extensively studied environmental response pathway in \textit{S. cerevisiae} \cite{Magasanik2005}. The low nitrogen signal begins when a depletion of cellular nitrogen triggers the protein Ure2p to form a prion aggregate, [URE3], and fall out of solution \cite{Wickner1994}. Soluble Ure2p binds the GATA-factor TFs Gat1p and Gln3p thereby sequestering them in the cytosol. When the aggregation of [URE3] is triggered, Gat1p and Gln3p are released and imported into the nucleus to trigger a regulatory cascade \cite{Zhang2018}. Gat1p and Gln3p work in complex with themselves and with the other zinc-finger TFs Dal80p and Gzf3p to promote expression of enzymes to break down alternative nitrogen sources. Much of what is known about this pathway has been studied in \textit{S. cerevisiae}, but the sister species \textit{S. paradoxus} also has a low nitrogen response \cite{Orlic2010}. These species additionally form F1 hybrids, allowing the assessment of the low nitrogen response in a mixed regulatory environment. 

Here we evaluate the effects of 46 TF knockouts in \textit{S. cerevisiae}, \textit{S. paradoxus}, and their F1 hybrid in the context of the low nitrogen response. We first identify the sets of genes in each of the three wildtype genotypes that alter expression in response to a low nitrogen signal. We find the largest category of nitrogen-responsive genes were conserved between the three genotypes, but these made up a minority of all gene responsive genes (32 \%). This set was enriched for Gene Ontology terms compared to the non-conserved sets. While all three genotypes had sets of genes that were unique, we found that the hybrid regulatory environment could recapitulate nearly every gene expression response that was shared between the two parental species, with the exception of only 8 out of 404 genes. For responses that were not conserved between parental species, the hybrid regulatory environment showed strong dominance for a single parental species' regulatory response shape. When we examined how 46 TF knockouts affected each genotype's low nitrogen response, we indeed found that different knockouts affected different genes and that only 6\% were conserved effects between all three genetic backgrounds. While they did not affect the same genes, we found these effects of the knockouts had a common theme. Specifically the effects tended to reduce the variability of the expression response to low nitrogen compared to wildtype. Taken together, our results demonstrate how characterizing environmental expression responses in wildtype genetic backgrounds can help contextualize the effects of TF knockouts in the same backgrounds.

\section{Results}

\subsection{Characterizing the wildtype Low Nitrogen response in all three genetic backgrounds}

To characterized the Low Nitrogen gene expression response in our three genetic backgrounds, we analyzed 3' tag-seq transcriptomic data in wildtype \textit{S. cerevisiae} (Scer), \textit{S. paradoxus} (Spar), and their F1 hybrid (Hyb) during the transition from a rich growth media to a nitrogen-poor growth media. These data were generated in Krieger et al. 2020 \cite{Krieger2020}, and include mRNA libraries from three timepoints in each of the three genetic backgrounds. The first timepoint was sampled after 6 hours of growth in rich media and immediately prior to transfer to low nitrogen media. As timepoints are named in relation to the nitrogen stress condition, this first timepoint is called ``0 hours, YPD." The second timepoint was collected after 1 hour of growth in the low nitrogen media (``1 hour, LowN") and the third after 16 hours of growth (``16 hours, LowN"). For each timepoint in each wildtype genetic background, the number of replicate samples ranged from 27 (Scer 1 hour, LowN) to 35 (Hyb any timepoint).

We first asked which genes responded to the Low Nitrogen environment with significant changes in expression. We considered a gene to respond to Low Nitrogen if 95\% of its counts at the second and/or third timepoint were not within 95\% of the count distribution established in the first timepoint. Depending on which timepoints this non-overlapping expression occurred at, we assigned each gene to a different response category (Fig \ref{fig:wt}A). If no pair of timepoints has less than 5\% overlap in its counts, the gene is assigned to the ``none" category. If the 1 hour timepoint has non-overlapping expression with the 0 hour timepoint, the gene is assigned to the ``early" category.  If the 16 hour timepoint has non-overlapping expression with the 1 hour timepoint, the gene is assigned to the ``late" category. If a gene can be considered both an ``early" and a ``late" responder, it is assigned to the ``both" category. Finally, if the 16 hour timepoint has non-overlapping expression with the 0 hour timepoint, but the gene cannot be considered an ``early" or ``late" responder, the gene is assigned to the ``gradual" category.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=0.8\textwidth]{chapters/figures/Networks/Fig_WT.png}}
    \caption{\textbf{Wildtype responses to Low Nitrogen in three genetic backgrounds.} A) Possible Low Nitrogen response categories for an example gene. Black points and boxplots represent the wildtype expression counts for a hypothetical gene. Brackets with a ``*" indicate pairs of timepoints with 95\% non-overlaping expression counts (the point distributions are not drawn to reflect this). Brackets with a ``n.s." indicate pairs of timepoints with $>$5\% overlapping expression counts. B) Number of genes in each response category in each genetic background. C) Venn diagram of which responsive genes (any response category) are common to multiple genetic backgrounds. D) Number of genes in each response category for only the genes that are responsive in all three genetic backgrounds.}
    \label{fig:wt}
\end{figure}

As expected, we found that the most genes in each genetic background belonged to the ``none" category and the fewest belonged to the ``both" category (Fig \ref{fig:wt}B). The ``gradual'' category was the second most prevalent in all three backgrounds. The other three response categories varied slightly in their proportions between genetic backgrounds. Specifically, Scer had more genes in the ``early'' category, while Spar had more genes in the ``late'' category. Hyb had the most responsive genes overall, but the proportions more reflected Scer's early response rather than Spar's late response.

We next asked whether the same genes were responding to Low Nitrogen in all three genetic backgrounds. We considered genes to be shared between backgrounds if a one-to-one ortholog was found to be responsive in both backgrounds, regardless of the response category it belonged to. We found that 32\% of responsive genes (396 genes) were responsive in all three genetic backgrounds, which was the largest category in the Venn diagram (Fig \ref{fig:wt}C). By far the smallest category were genes that were responsive in both Scer and Spar but not Hyb, which comprised only 8 genes. Significant numbers of genes were responsive in only one genetic background (58 in Spar, 131 is Scer, and 230 in Hyb), and significant numbers of genes were responsive in Hyb and one parental species (128 shared with Spar, 287 shared with Scer). The response categories of the 396 conserved responsive genes reflected those of the entire genome---with Scer and Hyb having more ``early" responders and Spar having more ``late" (Fig \ref{fig:wt}D).

\subsection{Genes that respond to Low Nitrogen in all three genetic backgrounds have the strongest enrichment of gene ontology terms}

We reasoned that genes with conserved responses in all three backgrounds could be more likely to be involved in biological processes related to Low Nitrogen. To test this, we looked for Gene Ontology (GOslim) enrichment among genes with conserved reponsiveness and compared these to responsive genes in the 5 other categories of the Venn diagram. We found that the 396 conserved responsive genes had the most GO terms with significant enrichment, with 71\% of genes belonging to at least one of the 25 significant GO terms (Fig \ref{fig:GO}). The other two categories with appreciable GO term enrichment were genes that responded in both Hyb and Scer (17 terms, 50\% of genes belonging to at least one term) and genes that responded only in Hyb (10 terms, 42\% of genes belonging to at least one term). Categories including Spar had the least GO term enrichment (2 for Hyb and Spar, 3 for Spar only). We found no GO term enrichment for genes shared between parents but not hybrid. but for reference the 8 genes in this category are: Cha1, Rad59, Snq2, Hmf1, Vhr2, Erg25, Src1, and Mlo1.
\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Networks/Fig_GOterm.png}}
    \caption{\textbf{Genes that are responsive to Low Nitrogen in all three genetic backgrounds have the most gene ontology enrichment} A) Same Venn diagram as in Fig \ref{fig:wt}C now pseudo-colored by category. B) Number of GO terms with significant enrichment for genes in each of the Venn diagram categories. Above each bar is the number of genes belonging to at least one of those terms as a fraction of total number of genes in the category.}
    \label{fig:GO}
\end{figure}

We next asked what GO terms were associated with conserved, Scer-Hyb, or Hyb responses. For brevity, we excluded GO terms that related to ribosome biogenesis (14 terms and 91 genes in conserved, 14 terms and 54 genes in Scer-Hyb, and 9 terms and 35 genes in Hyb). The remaining GO terms are presented in table \ref{tab:goslim_conserved} (conserved), table \ref{tab:goslim_ScerHyb} (Scer-Hyb), and table \ref{tab:goslim_Hyb} (Hyb) below. We found enrichment in expected categories related to metabolic processes of nitrogen-containing compounds, but also in less-expected categories, such as cytoplasmic translation, respiratory chain complexes, and 7/21 components of the eisosome. Full GO term results are available in supplementary tables in Appendix \ref{chpt:networkGO}.


% latex table generated in R 4.5.0 by xtable 1.8-4 package
% Tue Jul 22 16:43:23 2025
\begin{table}[H]
\centering
\begin{tabular}{rlrrl}
  \hline
 & GOslim term & group count & genome count & exact pval \\ 
  \hline
1 & amino acid metabolic process & 43 & 174 & 5.74e-13 \\ 
  2 & amino acid transport & 17 & 43 & 3.083e-09 \\ 
  3 & biosynthetic process & 144 & 1011 & 1.181e-16 \\ 
  4 & carbohydrate metabolic process & 19 & 117 & 0.0008668 \\ 
  5 & cytoplasmic translation & 56 & 97 & 4.808e-39 \\ 
  6 & eisosome & 7 & 21 & 0.0005381 \\ 
  7 & oligosaccharide metabolic process & 6 & 17 & 0.0009671 \\ 
  8 & oxidoreductase activity & 53 & 287 & 2.218e-10 \\ 
  9 & small molecule metabolic process & 89 & 581 & 6.317e-12 \\ 
  10 & structural molecule activity & 69 & 268 & 2.026e-21 \\ 
  11 & sulfur compound metabolic process & 17 & 87 & 0.000172 \\ 
  12 & translation & 56 & 194 & 4.939e-20 \\ 
  13 & transmembrane transporter activity & 43 & 312 & 4.394e-05 \\ 
  14 & vacuole & 36 & 254 & 0.0001036 \\ 
   \hline
\end{tabular}
\caption{GOslim terms not related to ribosome biogenesis that were significantly associated with genes that responded to Low Nitrogen in all three backgrounds}
    \label{tab:goslim_conserved}
\end{table}

% latex table generated in R 4.5.0 by xtable 1.8-4 package
% Tue Jul 22 16:43:33 2025
\begin{table}[H]
\centering
\begin{tabular}{rlrrl}
  \hline
 & GOslim term & group count & genome count & exact pval \\
  \hline
1 & cellular respiration & 13 & 59 & 1.018e-05 \\ 
  2 & cytoplasmic stress granule & 16 & 94 & 3.22e-05 \\ 
  3 & cytoplasmic translation & 21 & 97 & 2.585e-08 \\ 
  4 & generation of precursor metabolites and energy & 23 & 118 & 4.528e-08 \\ 
  5 & mitochondrial respiratory chain complex III & 7 & 9 & 3.898e-08 \\ 
  6 & mitochondrial respiratory chain complex IV & 5 & 10 & 8.624e-05 \\ 
  7 & multi-eIF complex & 5 & 10 & 8.624e-05 \\ 
  8 & nucleolus & 25 & 225 & 0.0003823 \\ 
  9 & response to starvation & 7 & 28 & 0.0005291 \\ 
  10 & RNA polymerase I complex & 5 & 14 & 0.0005732 \\ 
  11 & small molecule metabolic process & 49 & 581 & 0.0008408 \\ 
   \hline
\end{tabular}
\caption{GOslim terms not related to ribosome biogenesis that were significantly associated with genes that responded to Low Nitrogen in Scer and Hyb but not Spar}
    \label{tab:goslim_ScerHyb}
\end{table}

% latex table generated in R 4.5.0 by xtable 1.8-4 package
% Tue Jul 22 17:28:03 2025
\begin{table}[H]
\centering
\begin{tabular}{rlrrl}
  \hline
 & GOslim term & group count & genome count & exact pval \\
  \hline
1 & generation of precursor metabolites and energy & 16 & 118 & 3.796e-05 \\ 
  2 & \thead{nucleobase-containing small molecule metabolic process} & 19 & 176 & 0.0001753 \\ 
  3 & nucleolus & 28 & 225 & 2.711e-07 \\ 
  4 & oxidoreductase activity & 25 & 287 & 0.0005341 \\ 
  5 & small molecule metabolic process & 43 & 581 & 0.0002391 \\ 
   \hline
\end{tabular}
\caption{GOslim terms not related to ribosome biogenesis that were significantly associated with genes that responded to Low Nitrogen only in Hyb}
    \label{tab:goslim_Hyb}
\end{table}

\subsection{Parental gene expression patterns show strong dominance in the hybrid}

Thus far the Low Nitrogen response in Hyb has been seen to more closely resemble the response in Scer rather than Spar. This is evidenced both in terms of how many genes respond early versus late to Low Nitrogen and how many of those genes are common to both Scer and Hyb. We therefore asked if the expression patterns of individual genes would also more closely resemble Scer expression versus Spar. To calculate the similarity of the expression pattern of each gene, we first calculated the average expression of the gene among replicates within a genetic background at each timepoint to produce an expression vector of length 3. We then took the correlation of pairs of these expression vectors from Hyb and either parent---cor(Scer, Hyb) and cor(Spar, Hyb)---as our measure of expression pattern similarity.

\begin{SCfigure}
    \caption{\textbf{Hybrid expression tends to be strongly correlated with at least one parental species}. A) Correlation of hybrid expression vector (averaged expression among replicates at each timepoint) with the expression vectors of each parent. Only genes that responded to Low Nitrogen in at least one background are included. B) Same correlations as A, now including genes that were not considered responsive to Low Nitrogen but excluding genes that were highly correlated with both parents (cor(Scer, Hyb) $>=$ 0.8 and/or cor(Spar, Hyb) $>=$ 0.8). B') Inset of B for genes that were highly correlated with both parents (cor(Scer, Hyb) $>=$ 0.8 and/or cor(Spar, Hyb) $>=$ 0.8.}
    \fbox{\includegraphics[width=0.7\textwidth]{chapters/figures/Networks/Fig_HybridDominance.png}}
    \label{fig:dominance}
\end{SCfigure}

We found that genes that responded to Low Nitrogen had strong expression correlation ($>$ 0.8) between Hyb and one or both parents (Fig \ref{fig:dominance}A). We surprisingly did not find a tendency for more genes in Hyb to be correlated with Scer versus Spar. We instead see a relatively even split of genes correlated with either parent. Four genes were marked exceptions to this tendency: Exg2, Get3, YOR338W, and Ura4 all had hybrid expression that was uncorrelated with either parent (labeled in Fig \ref{fig:dominance}A). 

We next asked if this pattern of hybrid expression being correlated with at least one parent was also seen when we included genes that weren't Low Nitrogen responsive. For plotting, we excluded about half (2918/5359, 54\%) of genes that had strong expression correlation ($>$ 0.8) between Hyb and both parents, as these genes showed minimal expression pattern variation between genetic backgrounds. The remaining 2421 genes showed a similar distribution as the genes responsive to Low Nitrogen---genes tended to be strongly correlated between the hybrid and one parent (Fig \ref{fig:dominance}B). The pattern was not as stark, however, as 679/5359 (13\%) of genes had hybrid expression that was lowly correlated with both parents (cor(Scer, Hyb) $<$ 0.8 and cor(Spar, Hyb) $<$ 0.8), when previously only 10/1238 (0.8\%) of the Low Nitrogen responsive genes were in this category. Genes that were highly correlated between Hyb and both parents (cor(Scer, Hyb) $>=$ 0.8 and cor(Spar, Hyb) $>=$ 0.8) also tended to have one parent they were more closely correlated with, although the pattern was again not as stark (Fig \ref{fig:dominance}B').

\subsection{Transcription factor knockout lines do not have the same effect on gene expression between genetic backgrounds}

To characterize the effects of TF knockouts in each genetic background, we estimated the significant changes in gene expression in our 46 TF knockout lines relative to wildtype. We used DESeq2 \cite{Love2014} to identify genes with significant changes in expression in each knockout line relative to wildtype. Each knockout line was measured in two replicates, which gives us much lower power to detect expression differences that between wildtype samples, which each have about 30 replicates. To assess whether this imbalance in replicates led to false positives, we fit two additional DESeq2 models to 2 sets of 46 ``sham'' knockouts. These sham knockout lines each consisted of 2 replicates randomly sampled from wildtype gene counts.

We detected substantially more differentially expressed genes in the TF knockout lines versus the sham controls (Fig \ref{fig:tfdel}A). As previously reported \cite{Liu2024, Li2025}, we found that the majority of TF knockout effects were not shared between genetic backgrounds (Fig \ref{fig:tfdel}B). We considered an effect to be shared if the same gene changed expression significantly in the same TF knockout line in two or more backgrounds. Even with the least stringent definition of sharing---the affected gene could change expression in different directions and at different timepoints and still be considered shared---71\% of effects were unique to a single genetic background. Within the same genetic background, the different TF knockouts tended to affect different sets of genes, as most genes were only affected by 0, 1, or 2 TF knockouts (\ref{fig:tfdel}C).

\begin{figure}
    \fbox{\includegraphics[width=0.9\textwidth]{chapters/figures/Networks/Fig_TFdel.png}}
     \caption{\textbf{Differentially expressed genes in TF knockouts relative to WT in each of the three backgrounds}. A) Number of total effects across all TF knockouts, genes, and timepoints for each genetic background. Results using expression data of TF knockout lines are compared to two ``sham'' datasets where TF knockout expression is replaced with randomly sampled wildtype counts. B) Venn diagram of the number of effects that were shared between backgrounds. We considered an effect to be shared if the same gene was differentially expressed in the same TF knockout line regardless of timepoint of direction of expression change. C) Number of TF knockout lines in which each gene was differentially expressed in each background.}
    \label{fig:tfdel}
\end{figure}

\subsection{TF knockouts have widespread allele-specific effects in the F1 hybrid background}

To be comparable with the parental samples, we have thus far looked at gene expression in Hyb as the sum of both its Scer and Spar alleleic expression. To assess if the two parental alleles in the F1 hybrid tend to display different responses in the same TF knockout line, we asked whether DESeq2 models that were only given hybrid counts for the Scer allele (Hyc) or the Spar allele (Hyp) tended to detect the same genes as being differentially expressed. We found that Hyb log2 fold changes tended to be well correlated with the mean log2 fold change of Hyc and Hyp combined ($R^2$ = 0.82). When one hybrid allele was detected as differentially expressed in a TF knockout line, the other allele was also affected in 27\% of cases (Fig \ref{fig:alleleic}A). This lack of similarity between alleles was also reflected in the magnitude and direction of fold changes, as we found very little correlation between Hyc and Hyp log2 fold changes among our 5359 total genes ($R^2$ = 0.08). This low correlation was only marginally improved when we filtered for the cases where at least Hyc or Hyp was detected as significant ($R^2$ = 0.10).

\begin{figure}
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Networks/Fig_Alleleic.png}}
     \caption{\textbf{Hybrid alleles tend to respond differently in the same TF knockout line}. A) Number of TF knockout effects that affected one or both parental alleles in the F1 hybrid. B) Log2 fold changes of the Hyc vs Hyp allele for effects that were significant in both hybrid alleles. B' is an example of a black point, where log2 fold changes were in the same direction. B'' is an example of a red point, where log2 fold changes were in the opposite direction. C) Log2 fold changes of the Hyc vs Hyp allele for effects that were only significant in one hybrid alleles. C' is an example of a red point, where only the Hyc allele is significant. C'' is an example of a green point, where only the Hyp allele is significant}
    \label{fig:alleleic}
\end{figure}

To investigate why so so many TF knockout effects were allele-specific, we looked more closely at the two subsets of genes with allele non-specific or allele-specific effects. In the 27\% of cases where both alleles were detected as differentially expressed in the same TF knockout, the fold changes tended to be in the same direction---either both positive or both negative log2 fold changes (Fig \ref{fig:alleleic}B, example gene in \ref{fig:alleleic}B'). A distinct subset of genes in this category, however, had log2 fold changes in opposite directions (red points in Fig \ref{fig:alleleic}B, example gene in \ref{fig:alleleic}B''). In the remaining 73\% of cases where only one allele was significant, In both subsets, we noticed that the direction of response tended to be negative (Fig \ref{fig:alleleic}B and C, marginal distributions). That is, the affected gene tended to have lower expression in the TF knockout line versus wildtype.

\subsection{Transcription factor knockout lines have reduced expression variance compared to wildtype}

We next investigated whether this knowledge of the wildtype Low Nitrogen response would give us new insights into the widespread pleiotropic expression effects of TF knockouts. We first asked whether a gene was more likely to be differentially expressed in a TF knockout if they were in the dominant or recessive genetic background. Based on the distribution in \ref{fig:dominance}B, we assigned genes to their dominant parent based on which parental expression was more strongly correlated with the hybrid, excluding the genes that had less than a correlation of 0.8 with at least one parent. We found a slight bias for the dominant parent's background to be affected, with 3234/6257 (52\%, binomial p-value $<$ 0.004) of the parental and 4474/8666 (52\%, binomial p-value $<$ 0.002) of the hybrid alleleic effects occurring in the dominant background.

We lastly investigated whether the shape of a gene's wildtype expression could inform the direction of its differential expression in a TF knockout. As there were three timepoints in the experiment, we could unambiguously identify the timepoint where each gene had its minimum, middle, and maximum expression level (Fig \ref{fig:variance}A). We call these three categories the three ``stages'' of expression and note that each stage can occur at any of the three timepoints. We next asked whether the stage a gene was affected by a TF knockout affected the direction of its fold change. At all three stages, we saw that genes were more likely to have a negative than a positive effect from a TF knockout (Fig \ref{fig:variance}B). We found that the highest likelihood of a positive effect (i.e. gene expression increases in the TF knockout relative to wildtype) occurred when the gene was at its minimum expression. We found that the highest likelihood of a negative effect (i.e. gene expression decreases in the TF knockout relative to wildtype) occurred when the gene was at its maximum expression. The middle expression produced mixed results, with a larger proportion trending towards positive effects in Scer than in either Spar or Hyb.

\begin{figure}
    \fbox{\includegraphics[width=0.9\textwidth]{chapters/figures/Networks/Fig_Variance.png}}
     \caption{\textbf{Effects of TF knockouts tend to oppose wildtype expression extremes}. A) Illustration of a hypothetical gene in one genetic background. The timepoints showing min, mid, and max expression are defined by wildtype expression. Example of a significant TF knockout effect for a hypothetical TF is depicted as occurring at the mid timepoint and with a positive log2 fold change. B) Actual log2 fold changes of significant effects across all TFs, divided by which genetic background and expression level (min, mid, or max) was affected.}
    \label{fig:variance}
\end{figure}

\section{Discussion}

In this study, we investigated whether characteristics of the wildtype expression response to low nitrogen could inform the largely unpredictable expression effects in TF knockout lines. We found that 32\% of genes that were responsive to low nitrogen had conserved responses between our three genetic backgrounds. This conserved set had the strongest enrichment of gene ontology terms of any set, suggesting these genes were more likely than non-conserved groups to be involved in shared processes.

We observed a markedly different pattern, however, when assessing the conservation of gene expression effects of TF knockouts. Here, the identities of genes affected by each TF knockout had little overlap between backgrounds. Even between hybrid alleles, most effects were different. These allele-specific effects suggest that \textit{cis}-regulatory variation may modulate the effect of a TF knockout. Many genes with expression that is sensitive to variation in \textit{trans} later accrue compensatory changes in \textit{cis} \cite{Metzger2017, Metzger2019, Tsouris2024}. One possible explanation for the high proportion of allele-specific TF effects in hybrids is that these genes have above-average sensitivity to \textit{trans}-regulatory variation. While compensatory \textit{cis}-regulatory changes may restore an optimal expression level in the genetic background they evolved in, a change in genetic background may instead exacerbate their effects in the sub-optimal direction.

Among genes that showed low wildtype expression correlation between parental orthologs, we observed a striking pattern of dominance in the F1 hybrid. Specifically, the majority of genes had hybrid expression of both alleles that more strongly correlated with one parent than the other. This dominance suggests that one or a combination of \textit{trans}-acting factors may be present in the dominant parent's background and absent in the recessive parent's background. The presence of that \textit{trans}-acting factor(s) may then be sufficient to recapitulate the same expression in the F1 hybrid, with little interference from the recessive parent's alleles. We additionally found that the dominant background was slightly more likely to exhibit a response in a TF knockout line. It is possible these TF knockout effects are being in part mediated through the presence of the dominant \textit{trans}-acting factor or combination of factors.

Lastly, we observed that while TF deletions affected different genes in different genetic backgrounds, the effects had a common theme. Specifically, TF knockout lines exhibited less variation in expression across timepoints. This was most notable at the timepoint where the affected gene had its maximum expression. At this maximum, TF knockouts overwhelmingly biased the gene towards lower expression that wildtype. Many questions remain as to why this reduction in variation was observed in general across so many TF knockouts. One possibility is that any reduction in regulatory complexity---in this case, yeasts possessing one fewer regulator than wildtype---will tend to reduce the range of expression levels possible among genes. As the minimum expression level of any gene is 0, the lower bound of this range is already restricted. The reduction in expression range would therefore be more likely to be observed on the upper bound. Many questions remain as to whether this reduction of variability is seen in other environmental responses, other genetic backgrounds, or if it scales with the extent of decline in regulatory complexity (i.e. deletions of increasing numbers of regulators).

\section{Data Availability}

Raw RNA sequencing data from Krieger et al. 2020 are available at NCBI BioProject database (https://www.ncbi.nlm.nih.gov/bioproject/) under accession number PRJNA592756. Code used for analyses in this chapter is available at https://wyldtype.github.io/Networks/.

\section{Methods}

\subsection{Mapping and quantifying transcriptome data}

Batch scripts used to align reads in this section are performed in the script $\texttt{cut\_map\_count\_tagseq.sbat}$, available in the Chapter \ref{chpt:plasticity} github repository (https://github.com/wyldtype/Redhuis2025). STAR v2.7.11b was used for both aligning and quantification. Prior to aligning, STAR was run in --runMode genomeGenerate with the following parameters: --genomeSAindexNbases 11 --sjdbGTFfeatureExon gene --sjdbGTFtagExonParentGene Name to generate reference genome indices.

Genome assemblies and gene annotations for \textit{S. cerevisiae} (S288C) and \textit{S. paradoxus} (CBS432) were downloaded from yjx1217.github.io/Yeast\_PacBio\_2016/data/ \cite{Yue2017}. Genome assemblies for \textit{S. kudriavzevii} (GCA\_947243775.1) and \textit{S. uvarum} (GCA\_027557585.1) were downloaded from NCBI. The RNAseq data from Fay et al. 2023 was pooled with samples including \textit{S. kudriavzevii} and \textit{S. uvarum} and therefore needed to be mapped to a concatenated genome that included all four species. Only reads for the \textit{S. cerevisiae} and \textit{S. paradoxus} alleles were quantified and included in this study.

50bp 3' tag-seq reads from Krieger et al. 2020 were mapped to the \textit{S. cerevisiae} and \textit{S. paradoxus} concatenated genome using STAR with the following parameters: --alignEndsType EndToEnd --outFilterScoreMinOverLread 0  --outFilterMatchNminOverLread 0 --outFilterMismatchNmax 4  --scoreGap -10. STAR was run in --quantMode GeneCounts to concurrently count reads within 500 base pairs of the transcription end site ($\pm$500 TES). An average of 60\%, or 1 million reads, were uniquely mapping per sample. Only uniquely mapping reads overlapping $\pm$500 of an annotated TES were quantified.

\subsection{Normalizing read counts}

Count data was normalized to counts per million using the same procedure as for the 50bp 3' tag-seq data in Chapter \ref{chpt:plasticity}. Normalizing one gene's expression in one sample is as follows:

$$ gene_i \text{ (counts per million)} = \frac{\text{ reads mapped to } gene_i}{\text{library size}}*10^6 $$.

\subsection{Differential expression in transcription factor deletion lines}

To detect genes with differential expression in response to each transcription factor deletion, we estimated log2 fold change using DESeq2. We chose DESeq2 over the generalized linear models used in Chapter \ref{chpt:plasticity} because each transcription factor deletion only had two replicate samples. DESeq2 estimates expression variation across many genes thereby increasing power to detect DE genes when few replicate samples are available \cite{Love2014}. We generated separate DESeq data sets for each timepoint in each genetic background. Prior to model-fitting, we removed samples for any TF knockout that did not have at least two replicates (CBF1, GCN4 and YAP1 in Hyb, ZAP1 in Scer). For each TF knockout in each timepoint in each genetic background, we extracted from the DESeq2 model the p-value and estimated log2 fold change in expression when contrasting wildtype and TF knockout genotype.  We considered a gene to be significantly differentially expressed at a certain timepoint in a certain TF knockout line if its p-value was $\leq$0.05 (DESeq2 automatically applies false discovery correction) and magnitude of log2 effect size was $\geq$2. We repeated the DESeq2 model fitting two additional times with two ``sham'' datasets where the only difference from the real data is that expression counts for every TF knockout sample are sampled randomly from wildtype counts of the same genetic background at the same timepoint.
