\chapter{Effects of transcription factor knockouts on signal propagation through regulatory networks}
\label{chpt:networks}

\section{Introduction}
\label{sec:intro}

Signal propagation through gene regulatory networks is a vital process through which organisms respond to changes in their environment. Signaling begins when environment-sensing proteins alter their structure due to a change in cellular state, thereby triggering a gene regulatory cascade \cite{Bahn2007}. This cascade ultimately results in the upregulation of structural genes whose products play important roles in adjusting to the environmental change. For example, upregulation of enzymes involved in breaking down alternative carbon sources in response to a glucose shortage \cite{Gancedo1998}. While both the environmental sensors and the structural genes tend to be well conserved between species \hl{BIGSWING}, species vary to a surprising degree in how the signal transduces through the regulatory network \hl{Cite comparative signaling}. This variation has important implications for how regulatory networks evolve. In particular, how much neutral space is available in signaling networks structure for populations to explore over evolutionary time \cite{Payne2015}.

The intermediaries in a signal transduction response are transcription factors (TFs). A common strategy to assess how genes are connected in a regulatory network is to examine which genes have altered mRNA abundance in response to repression of specific TFs. Recent studies in yeast have taken advantage of gene editing tools and a relatively small set of TFs (169 recorded in the Yeastract database \cite{Hahn2011}) to compare how the same set of TF knockouts affect gene expression in different genetic backgrounds. Surprisingly, these studies have found that the majority of differentially expressed genes in each TF knockout line are non-overlapping sets. In fact, the sets of differentially expressed genes between different \textit{S. cerevisiae} strains have as much overlap as would be expected by random sampling of gene sets in each strain \cite{Li2025}. The majority of effects also tend to be measured in genes located far from where the TF bound \cite{Mahendrawada2025}. The minority \hl{percent} of genes with conserved effects between strains tend to remain conserved between more distantly related species \cite{Liu2024}. Taken together, these studies suggest that this lack of predictability in which genes will be differentially expressed in response to a given TF perturbation is the result of widespread pleiotropic expression responses that change rapidly with evolutionary time. 

Although these unpredictable pleiotropic effects of TF knockouts are likely of little phenotypic consequence for the organism, they are intriguingly stable within a specific genotype. These effects are highly repeatable in multiple measurements of the same genetic background and can be detected quickly after the onset of TF perturbation \cite{Mahendrawada2025}. 

The Low Nitrogen response pathway in yeast


\section{Results}
\label{sec:results}