\chapter{Plasticity of regulatory networks}
\label{chpt:plasticity}

\section{Abstract}

Organisms often cope with changes in their environments by modifying gene expression levels, which can affect their cellular function. This plasticity in gene expression arises when cells sense a change in their environment and alter the activity or availability of \textit{trans}-regulatory factors, which interact with each gene’s \textit{cis}-regulatory sequences to determine its expression. To understand how regulatory networks controlling plasticity in gene expression evolve, we used RNAseq data from the baker’s yeast \textit{Saccharomyces cerevisiae}, its close relative \textit{Saccharomyces paradoxus}, and their F1 hybrids collected at multiple time points following the transfer of cells from standard laboratory conditions to five different environments (low phosphorus, low nitrogen, hydroxyurea shock, heat stress, and cold stress. We also looked at how gene expression changes during the transition from log phase to stationary phase. In all six datasets, we compared gene expression levels between  \textit{S. cerevisiae} and  \textit{S. paradoxus} and asked how expression plasticity has diverged. We then tested for divergence in expression plasticity between the two species-specific alleles in F1 hybrids, allowing us to disentangle the effects of divergence in \textit{cis}- and \textit{trans}-regulation. In all 6 environments, we found at least 100 genes showing divergence in expression plasticity between species. Most cases were unique to a single environment and attributable primarily to  \textit{trans}-regulatory divergence. Our work demonstrates how comparing species in diverse environments and multiple timepoints can reveal hidden dimensions of gene expression divergence.

\section{Introduction}

Organisms live in constantly changing environments and have evolved to cope with these environmental changes through phenotypic plasticity. Here we define plasticity as the ability of a single genotype to produce alternate phenotypes under different environmental conditions. Plasticity is controlled at the molecular level by changes in the regulation of gene expression. This regulation depends upon interactions between \textit{cis}-acting DNA sequences (e.g., enhancers, promoters) and \textit{trans}-regulatory RNAs and proteins (e.g., transcription factors). Evolutionary divergence in either \textit{cis}-regulatory sequences or sequences encoding \textit{trans}-regulatory factors can alter the regulation of gene expression. Changes in expression plasticity between populations and species can be important sources of ecological diversification \cite{HodginsDavis2009, Lasky2014, Ghalambor2015, Makinen2018, Ballinger2023}. Studies on expression plasticity evolution have typically focused on understanding responses to a specific environmental shift (e.g. response to shift in temperature). In reality, organisms experience a range of different environmental shifts over their lifetimes. Understanding how the same genetic divergence affects expression plasticity across a range of environments is therefore essential to understanding how regulatory evolution proceeds.
Expression plasticity begins when a shift in environment alters a cellular state, for example via depletion of a nutrient or a change in temperature. This altered cellular state then triggers a change in gene expression. Expression plasticity is not simply a gene being ``on” or ``off” in different environments. Rather, the dynamics of how quickly expression is activated or repressed in response to a change in environment is the realization of expression plasticity \cite{Hager2009, Rivera2021, Wagh2023}. Divergence in expression plasticity, therefore, is best thought of as shifts in these expression dynamics between species \cite{Koster2015}. These shifts can either be gradual, for example if a plastic response changes gradually between populations over evolutionary time \cite{Makinen2018}. Alternatively, expression plasticity can change dramatically between closely related species, for example a gain of function in a regulator \cite{Soppe2000}. Both types of plasticity divergence are expected to indirectly affect the expression of many genes, due to the interconnectedness of the gene regulatory network. Importantly, however, this indirect expression divergence in many genes will only be observed in the environments that expose the causative plasticity divergence.
Expression divergence between two species is either environment-specific (plastic) or environment-robust. Here we refer to environmentally-robust expression divergence as expression level divergence and environment-specific expression divergence as expression plasticity divergence. Level and plasticity divergence are confounded when expression is measured in one environment at one timepoint (Fig \ref{fig:conjectogram}A). Most studies that contrast level and plasticity divergence resolve this issue by using expression data from multiple environments (e.g. \cite{Dayan2015, Ghalambor2015, Ballinger2023} (Fig \ref{fig:conjectogram}B-C). Measuring over multiple timepoints during a response to a change in environment can capture the same difference in level vs plasticity while additionally providing insight into the dynamics of the expression response (Fig \ref{fig:conjectogram}D-E).

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/IntroConjectogram.png}}
    \caption{A) One gene diverging in expression between two species, measured in one environment. B) Expression of the same gene measured in multiple environments, if the gene is diverging in expression level. C) Expression of the same gene measured in multiple environments, if the gene is diverging in expression plasticity. D) Expression of the same gene measured at multiple timepoints in response to two environmental shifts, if the gene is diverging in expression level. E) Expression of the same gene measured at multiple timepoints in response to two environmental shifts, if the gene is diverging in expression plasticity.}
    \label{fig:conjectogram}
\end{figure}

Studies contrasting level and plasticity divergence have found they are largely evolving independently: divergence in mean expression level does not predict divergence in environment-specific expression patterns \cite{Krieger2020}, and level and plasticity divergence tend to affect separate suites of genes \cite{Dayan2015}. This is consistent with these two properties tending to be caused by separate molecular mechanisms: expression divergence in \textit{cis} to the affected gene tends to be more environmentally-robust than expression divergence in \textit{trans} \cite{Tirosh2009, Krieger2020}. While interpreting the \textit{cis}-regulatory code underlying gene expression variation is an area of active research \cite{DeBoer2020, Avsec2021}, comparatively less is known about the more environmentally-plastic and predominantly \textit{trans}-regulatory variation. Is it a significant source of expression variation between species? What genes or pathways might exhibit it? What molecular mechanisms might contribute to it?
Here, we measure divergence in expression level and divergence in expression plasticity between all orthologous genes in two species of yeast: Saccharomyces cerevisiae and its close relative Saccharomyces paradoxus in six environments from two published datasets \cite{Krieger2020, Fay2023}. We find that in both datasets hundreds of genes were identified to be diverging in expression level, plasticity, or both properties in each environment. Despite being measured independently in each of the six environments, expression level divergence is consistent between environments and largely but not entirely consistent between strains. Expression plasticity divergence, in contrast, was unique to each environment. Each of the six environments revealed a different subset of at least 100 genes that have diverged in their expression plasticity. Our work presents a comprehensive survey of the two types of gene expression divergence and demonstrates a simple and effective framework to compare both properties and gain insight into their molecular mechanisms.

\section{Results \& Discussion}

\subsection{Experimental overview}

To characterize the extent of divergence in expression level and expression plasticity between \textit{S. cerevisiae} and \textit{S. paradoxus}, we analyzed RNAseq data from both species at multiple time points in six different growth conditions (Fig \ref{fig:overview}): (A) growth to saturation in rich (YPD) media \cite{Krieger2020}, (B) progression through the cell cycle in rich media following synchronization with urea shock \cite{Lupo2021}, (C) transition to and growth in low nitrogen media from rich media \cite{Krieger2020}, (D) transition to and growth in low phosphorous media from rich media \cite{Krieger2020, }, (E) transition to and growth at 37C from 25C (heat shock) \cite{Fay2023}, (F) transition to and growth at 12C from 25C (cold shock) \cite{Fay2023}. In each case, total RNA was sampled in the starting condition and then at multiple time-points after the change in environment to measure how transcript abundance changes as yeast acclimate to the shift in environment. These data allowed us to determine how the dynamic expression response of yeast cells in multiple environments has diverged between species.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/ExperimentOverviewWithTimepoints.png}}
    \caption{\textbf{Datasets used in this study}}
    \label{fig:overview}
\end{figure}

In each of the six environmental datasets, we measured divergence in both expression level and expression plasticity between species (Fig \ref{fig:workflow}A). In each environment, we first filtered out genes that were lowly expressed in both species (average expression less than 30 counts per million among timepoints). This filter found, depending on the environment, 9-20\% of genes had low expression in both species (Fig. \ref{fig:workflow}B).

We define divergence in expression level as changes in expression that are seen consistently in all time points within an environment, whereas divergence in expression plasticity is defined as a difference in expression levels specific to a subset of time points within an environment. To measure divergence in expression level for each gene in each environment, we fit the generalized linear model $Y_{ij} = allele_i + timepoint_j$, where $Y_{ij}$ is the read count from the RNAseq data for species $i$ at timepoint $j$ (see Methods). The coefficient for the allele term is the estimated log2 fold change of \textit{S. cerevisiae}’s count relative to \textit{S. paradoxus}’, in a single environment, controlling for time-point. This measure of log2 fold change therefore describes the difference in expression level between species independent of any difference in expression plasticity (Fig \ref{fig:workflow}C). A gene was considered to have diverged in expression level if its log2 fold change had a magnitude greater than 1 and an associated Wald test p-value less than $1x10^{-5}$.
To examine expression plasticity, we grouped orthologous genes by hierarchical correlation clustering (Fig \ref{fig:workflow}C). To identify genes diverging in plasticity between species, we identified all instances where a one-to-one ortholog was placed in different clusters in both species (Fig \ref{fig:workflow}D, See Methods). Correlation clustering is independent of any constitutive changes in expression level because correlations are invariant to multiplying by a constant, i.e. log2 fold change. Prior to clustering, we assigned genes that had low expression variability among time-points to a low-variance cluster, as these genes will be weakly correlated with all other genes. We then clustered the remaining genes by hierarchical correlation clustering. Hierarchical clustering produces a single tree of all genes, and the number of clusters produced depends on where the user chooses to cut this tree. To determine how a chosen number of clusters affected our results, we compared results using 2, 3, or 4, clusters. We found that 97\% of the genes identified as changing clusters in the 2-cluster scheme were also identified as changing clusters in the 3 and 4-cluster scheme (Fig S\ref{fig:qc_clusters}). The 2-cluster scheme was the most conservative definition of plasticity divergence, with 80\% of the genes identified as changing clusters in the 3 or 4-cluster schemes also identified as changing clusters in the 2-cluster scheme. We chose to continue the analysis with the 2-cluster scheme, as this was the most conservative and simplest clustering scheme.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Workflow.png}}
    \caption{\textbf{Experimental Workflow}}
    \label{fig:workflow}
\end{figure}

Limiting to two clusters tended to capture genes that either increased or decreased expression in response to the environmental shift (Fig \ref{fig:clusters}). There was environment-to-environment variability in how this increase or decrease was realized. Genes in Heat and Cold Stress had their minimum or maximum average expression at a middle timepoint, whereas the other four environments had monotonic increases or decreases in expression. Genes In Hydroxyurea Shock and Low Nitrogen had mean expression level differences between clusters, while the other four environments did not. In all six environments, because the two clusters were based on grouping genes with the strongest correlation together, the average expression of each of the two clusters was strongly negatively correlated with each other. This results in the clusters appearing to be as close to mirror images of each other as possible given the genes they comprise.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Clusters.png}}
    \caption{\textbf{Expression plasticity clusters in each environment, average of both species.}}
    \label{fig:clusters}
\end{figure}

\subsection{Level and plasticity divergence before and after Diauxic Shift}

To characterize divergence in expression level and expression plasticity, we began by comparing both properties in a single environment. Results for the Diauxic Shift environment are used as an example in this section, but similar results were found in each of the six environments (Fig S\ref{fig:all_environments}). Growth to a saturated population size triggers the yeast metabolic response to the exhaustion of glucose, called the diauxic shift. In this environment, 579 genes were lowly expressed in both species and therefore excluded, leaving a total of 4286 genes. We found that in the Diauxic Shift environment, genes clustered into three expression plasticity clusters: 1) genes that increased expression at the diauxic shift, 2) genes that decreased expression at the diauxic shift, and 3) genes that did not change their expression at the diauxic shift (Fig \ref{fig:clusters}). In \textit{S. cerevisiae}, 1962 genes were included in the increasing cluster, 1353 genes in the decreasing cluster, and 971 in the static cluster (i.e., they were expressed similarly among time points). In \textit{S. paradoxus}, 2515 genes were part of the increasing cluster, 921 genes in the decreasing cluster, and 850 genes in the static cluster.
We noted significant differences between species in the number of genes belonging to each plasticity cluster, suggesting extensive divergence in expression plasticity as we defined in the previous section. We found that the largest portion of genes (2086/4286 genes, 48\%) had conserved expression level and expression plasticity between species (Fig \ref{fig:diauxic}A). A smaller portion (752/4286, 18\%) had diverged in expression level but not expression plasticity (Fig \ref{fig:diauxic}C). A similar portion (1005/4286, 23\%) had conserved expression level but had diverged in their expression plasticity (Fig \ref{fig:diauxic}D). As expected, we found the least frequent category was genes that had diverged in both expression level and expression plasticity (443/4286, 11\% Fig \ref{fig:diauxic}E, bottom right).
Our results generally agreed with prior findings that changes in expression level or plasticity are independent modes of gene expression divergence. There was slight enrichment of genes diverging in both level and plasticity in Diauxic Shift (Fisher Exact Test odds ratio 1.22, p < 0.001) and a slight paucity in of genes diverging in both level and plasticity in Heat Stress (Fisher Exact Test odds ratio 0.95, p < 0.04), but there was no significant relationship in the other 4 experiments. Furthermore, in 5/6 environments, genes with diverged plasticity had no significant difference in their average log2 fold changes compared to genes with conserved plasticity (two-tailed t.test, p > 0.05, Fig S\ref{fig:boxplots}). The exception was Cold Stress, but only when limiting to genes with higher expression level in \textit{S. cerevisiae}, suggesting the significant p-value could be a result of multiple testing.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/SaturatedGrowth.png}}
    \caption{\textbf{Separating divergence in expression level from divergence in expression plasticity in the Diauxic Shift environment.} A) Average expression of genes with conserved expression level and plasticity that increase expression at the diauxic shift, error ribbons indicate 95\% confidence in the mean. B) Proportional area plot of total number of genes in each of the 4 divergence categories. C) Average expression of genes that have higher expression level in S. cer and increase expression at the diauxic shift. D) Average expression of genes that increase expression in S. par and decrease expression in S. cer at the diauxic shift. E) Average expression of genes that have higher expression level in S. par and increase expression at the diauxic shift only in S. par.}
    \label{fig:diauxic}
\end{figure}

In all six environments, the proportions of genes diverging in expression level and expression plasticity were similar (Fig S\ref{fig:all_environments}). The proportion of genes diverging in expression level, however, was highly dependent on the chosen effect size threshold—without a threshold, roughly 3 times as many genes were diverging in expression level versus expression plasticity. Conversely, with a stronger threshold necessitating that a gene be expressed twice as high in one species to be considered diverging in level, roughly 10 times more genes were diverging in plasticity than level (Fig S\ref{fig:threshold}). Thus, the proportion of genes diverging in expression plasticity is most comparable to the proportion of genes with at least a moderate divergence in expression level (log2 fold change of 1.5 or greater between species).
We noted that the number of genes diverging in plasticity but not level was surprisingly high. In order for these genes to truly have no level divergence, the species that decreases expression over time in each experiment must have begun the experiment with higher expression. As every environment began in standard laboratory conditions, these genes would be expected to also show the same difference in starting expression in all six environments. We confirmed that these genes did indeed tend to show higher starting expression in the species that would decrease expression over time in all six experiments (Fig S\ref{fig:tp0}). These genes would appear to have consistent expression level divergence between \textit{S. cerevisiae} and \textit{S. paradoxus} if their expression was only measured in standard laboratory conditions.

\subsection{Similar portions of genes are diverging in level and plasticity in each environment}

We next asked whether similar patterns of level and plasticity divergence were seen in the other five environments. We found that the proportions of genes diverging in each category was roughly the same for each environment (Fig \ref{fig:proportions}A). In all cases, the largest portion of genes had conserved level and plasticity (solid grey in Figure \ref{fig:proportions}A), roughly equal numbers of genes had diverged level (striped grey in Figure \ref{fig:proportions}A) or plasticity (solid gold in Figure \ref{fig:proportions}A), and the smallest portion of genes had diverged level and plasticity (striped gold in Figure \ref{fig:proportions}A). For level divergence, the number of genes expressed higher in \textit{S. cerevisiae} than \textit{S. paradoxus} varied by environment (Fig \ref{fig:proportions}B). The Hydroxyurea Shock, Low Nitrogen, Heat Stress, and Cold Stress environments had over 50\% of genes with higher level in \textit{S. cerevisiae} whereas the Diauxic Shift and Low Phosphate environments had over 50\% of genes with higher level in \textit{S. paradoxus}.
Expression plasticity can be classified into three categories: 1) conserved plasticity, the gene remains in the same cluster for both species, 2) species-specific plasticity, the gene is in the low variance/zero cluster in one species but not the other, 3) plasticity reversal, the gene has switched from one non-zero cluster to the other non-zero cluster between species (Figure \ref{fig:proportions}C). Given that a plasticity reversal would mean a gene switching from increasing in one species to decreasing in the other species in response to the same environmental trigger, we expected this to be the least common category. We were surprised to find that in some environments (Hydroxyurea Shock and Low Phosphate especially), plasticity reversals were as common if not more common than species-specific plasticity (Figure \ref{fig:proportions}D).
It is important to note that two genes may have strongly negatively correlated expression and still be involved in the same biological process. For example, a repressor and an activator of non-fermentable carbon enzymes would both change expression at the Diauxic Shift, but in opposite directions. These plasticity reversals, however, are affecting the same one-to-one ortholog, and it is unlikely that hundreds of genes changed from having a repressing to an activating role in each environment over this evolutionary timescale. More likely is that many of these plasticity-reversing genes are co-regulated and are involved in the same biological process. To assess which biological processes might be associated with genes reversing plasticity in each environment, we performed a gene ontology enrichment analysis using GOslim \cite{Skrzypek2011}. All 12 gene groups (6 environments x 2 directions) had significant enrichment of at least one biological process (12/12 had Fisher’s exact test p $<$ 0.003). To summarize gene ontology results for each gene group, we combined related terms into larger term categories and present these summaries in Table \ref{tableGO}.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/EnvironmentalQuants.png}}
    \caption{\textbf{Similar portions of genes are diverging in level and plasticity in each environment.} A) Stacked barplot of number of level and plasticity diverging genes in each environment B) Ridgeline plot of log2 fold changes of each level-diverging gene in each environment. C) Procedure for identifying plasticity divergence in a single environment based on a gene’s cluster in each species D) Number of genes diverging in plasticity in each environment}
    \label{fig:proportions}
\end{figure}

\begin{table}[H]
\centering
\begin{tabular}{|l|c|c|l|}
\hline
{\bf Environment} & {\bf cluster Scer} & {\bf cluster Spar} & {\bf GOslim summary (fraction of genes)} \\ 
\hline
Diauxic Shift & increasing & decreasing & protein folding/modification, \\
&&& transport (23/53) \\ \hline
Diauxic Shift & decreasing & increasing & DNA repair, golgi/vesicle \\
&&& transport (103/284) \\ \hline
Hydroxyurea Shock & increasing & decreasing & respiration, metabolism (191/415) \\ \hline
Hydroxyurea Shock & decreasing & increasing & ribosome, tRNA processing (136/555) \\ \hline
Low Nitrogen & increasing & decreasing & protein folding, translation (22/51) \\ \hline
Low Nitrogen & decreasing & increasing & lipid/amino acid/tRNA synthesis (35/77) \\ \hline
Low Phosphate & increasing & decreasing & protein folding/modification,\\
&&& translation (118/391) \\ \hline
Low Phosphate & decreasing & increasing & ribosome, tRNA/small molecule \\
&&& synthesis (368/797) \\ \hline
Heat & increasing & decreasing & protein folding/modification, \\
&&& translation (118/391) \\ \hline
Heat & decreasing & increasing & transcription, small molecule \\
&&& synthesis (184/470) \\ \hline
Cold & increasing & decreasing & translation, ribosome, and nitrogen \\
&&& metabolism (118/246) \\ \hline
Cold & decreasing & increasing & translation, transport (38/101) \\ 
\hline
\end{tabular}
\caption{{\bf Gene ontology enrichment categories among plasticity-reversing genes}}
 \label{tableGO}
\end{table}

\subsection{Divergence in plasticity tends to be unique to single environments}

Having classified level and plasticity divergence separately in each environment, we next asked whether the same genes were diverging in each property in multiple environments. To assess the concordance of both properties across environments, we first asked how often genes belonged to the same divergence category (conserved, diverged plasticity, diverged level, or diverged level and plasticity) in multiple environments. We found that the divergence category in one environment was a poor predictor of what category the same gene would belong to in other environments (Fig \ref{fig:patterns}A). To determine whether a more continuous rather than categorical measurement would be a better predictor across environments, we also compared genes by their log2 fold change (Fig \ref{fig:patterns}B, D, and E) and expression correlation (Fig \ref{fig:patterns}C and F).
To measure log2 fold change, we selected the genes identified in one environment as diverging in level and measured their average log2 fold change in each of the six environments. We found that groups of genes had generally consistent fold change in other environments (Fig \ref{fig:patterns}D and E, example in G). Fold changes were more consistent within the same pair of Saccharomyces strains than between strains (Fig \ref{fig:patterns}D and E, compare Heat/Cold to other 4 environments).
To measure expression correlation, we followed a similar procedure (Fig \ref{fig:patterns}C). In one environment, we first identified the genes diverging in plasticity. We excluded genes that were part of the static cluster in either species because they will have low correlation with all other genes. We then measured the correlation of those same plasticity-diverging genes in other environments. We found that the majority of gene groups only had strongly negative correlations in the environment they were identified in (Fig \ref{fig:patterns}F, example in H).

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/EnvironmentalPatterns.png}}
    \label{fig:patterns}
\end{figure}

\begin{figure}
    \centering
    \caption{\textbf{Divergence in expression plasticity is more environment-specific than divergence in expression level.} A) Categorical heatmap of divergence category of each gene in each environment. Genes are ordered by their category in the top environment first then genes within each category are ordered in the next row in a nested fashion. B-C) Procedure for calculating average log2 fold change and average expression correlation of genes identified as divergent in each environment. D-E) Heatmaps of average log2 fold change of genes higher expressed in S. cer and S. par respectively. F) Heatmap of average expression correlation of any genes diverging in plasticity excluding low variance genes. G-H) Example average expression of groups of genes diverging in level and plasticity respectively. Red boxes indicate the environment the groups of genes were identified in.}
    \label{fig:patterns}
\end{figure}

\subsection{Expression plasticity divergence is primarily in \textit{trans} while expression level divergence is primarily in \textit{cis}}

We have determined that divergence expression plasticity between species is only seen in certain environments. We reasoned that these diverging genes must be regulated at least in part by environment-specific proteins. Genetic divergence causing these genes to diverge in expression could be in one of three places relative to these environment-specific regulators: 1) in coding sequences of these environment-specific proteins (coding, \textit{trans}), 2) in regulatory regions controlling expression of these environment-specific regulators (noncoding, \textit{trans}), or 3) in the DNA regions that these environment-specific regulators bind to (noncoding, \textit{cis}). To separate the third possibility from the first two, we analyzed the allele-specific expression of genes in the \textit{S. cerevisiae} x \textit{S. paradoxus} F1 hybrid in the same six environments. Expression differences in level or plasticity that persist between hybrid alleles must be caused by genetic variation in \textit{cis} to the affected gene (Fig 7A).
To assess what proportion of expression level divergence was due to \textit{cis}-regulatory divergence, we partitioned genes into two groups based on their magnitude of fold change between alleles in the F1 hybrid. Genes with high magnitude of fold change between alleles (|fold change| $>$ 1.5), we considered to be diverging in level predominantly in \textit{cis}. Genes with high magnitude of fold change between alleles (|log2 fold change| $\leq$ 1.5), we considered to be diverging in level predominantly in \textit{trans} (Fig \ref{fig:cistrans}A). We found that the majority of genes in all six environments belonged to the group with high fold change between alleles, suggesting the majority of expression level divergence is in \textit{cis} (Fig \ref{fig:cistrans}B). To visually confirm these results, we compared average expression between genes from each group (Fig \ref{fig:cistrans}D and E).
To assess what proportion of expression plasticity divergence was due to \textit{cis}-regulatory divergence, we partitioned genes into two groups based on their expression correlation between alleles in the F1 hybrid. Genes with low expression correlation between alleles (cor $<$ -0.25), we considered to be diverging in plasticity predominantly in \textit{cis}. The remaining genes (cor $\geq$ -0.25), we considered to be diverging in plasticity predominantly in \textit{trans} (Fig \ref{fig:cistrans}A). We found that the majority of genes in all six environments belonged to the group with higher expression correlation between alleles, suggesting the majority of expression plasticity divergence is in \textit{trans} (Fig \ref{fig:cistrans}C). To visually confirm these results, we compared average expression between genes from each group (Fig \ref{fig:cistrans}F and G).

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/CisTrans.png}}
    \caption{\textbf{Level divergence tends to be in \textit{cis} while plasticity divergence tends to be in \textit{trans}.} A) Expression level divergence in \textit{cis} or \textit{trans} can be separated based on high or low log2 fold change magnitude (absolute value) between hybrid alleles. Expression plasticity divergence in \textit{cis} or \textit{trans} can be separated based on high or low expression correlation between hybrid alleles B) Percents of genes with higher fold changes (\textit{cis}-diverging) versus lower fold changes (\textit{trans}-diverging) between hybrid alleles. C) Percents of genes with lower correlation (\textit{cis}-diverging) versus higher correlation (\textit{trans}-diverging) between hybrid alleles. D-G) Average expression of genes in the high and low portions of B and C, example environment shown is Diauxic Shift.}
    \label{fig:cistrans}
\end{figure}

\subsection{Environment-specific regulons can have divergent expression in alternative environments}

We found that expression level divergence tends to be robust across timepoints and environments and largely based in \textit{cis}. This is consistent with what has been previously seen when comparing gene expression at one timepoint in multiple environments (beginning with \cite{Tirosh2009}). Our findings extend this robustness to also apply to timepoints within an experiment. This timepoint-scale robustness suggests a possible molecular mechanism controlling expression level divergence. Specifically, that level divergence is caused by \textit{cis}-regulatory divergence affecting transcript abundance in any environmental condition that the gene is expressed in. We have further seen that different strains can have different sets of genes diverging in level, suggesting that expression level divergence can be quickly tuned in evolutionary time.
Divergence in expression plasticity, in contrast, appears to be largely an indirect effect of divergence in the \textit{trans}-regulatory molecular environment. These changes to the molecular environment between species are only revealed in response to specific environmental triggers. To better understand what exactly has diverged about the molecular environment between \textit{S. cerevisiae} and \textit{S. paradoxus}, we turned to the environmental sensing literature in Saccharomyces. To identify the proteins known to respond to each environmental shift in our dataset, we downloaded regulatory matrices of environment-specific regulators and their regulons from the yeastract database \cite{Teixeira2006, Teixeira2023}.
In each regulatory matrix, each row is a potential regulator and each column is a potential regulatory target. A 1 in any cell indicates that there is evidence of the regulator binding near that gene and that perturbation of the regulator affects its expression (either positively or negatively) in that environmental condition. All other cells containing regulator-target combinations with no known relationship in that environment are filled with 0. To separate environment-specific responses from general regulatory networks, we also included the regulatory matrix of un-stressed yeast in standard laboratory conditions. Importantly, all seven of these regulatory matrices were defined from \textit{S. cerevisiae} regulatory data, so it is possible that certain regulatory relationships are missing in \textit{S. paradoxus}, and other regulatory relationships may be missing if they are only found in \textit{S. paradoxus}. Yeastract regulatory matrices are updated by hand as new studies conducted in each environment are published. As a result of both the curation process and true yeast biology, the matrices vary in the number of regulatory relationships recorded in each environment (Fig \ref{fig:yeastract}A). We therefore consider these matrices to be an incomplete sampling of regulatory relationships in \textit{S. cerevisiae} in each of these environments. Importantly for our purposes, this sampling is independent from the expression data we have discussed in previous sections and thus provides an additional layer of regulatory information to cross reference with our analysis of plasticity.

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Yeastract.png}}
    \caption{\textbf{Regulatory network connections among each plasticity gene group.} A) In degree: number of incoming connections to each gene from environment-specific or un-stressed regulatory networks compared to the proportion of genes in each plasticity category. B) Out degree: number of outgoing connections from each gene from environment-specific or un-stressed regulatory networks compared to the proportion of genes in each plasticity category. C) visual representations of each environment-specific regulatory network. A larger version where regulator names are visible is available in Supplementary files. D) Number of regulatory targets per environment-specific regulator in B).}
    \label{fig:yeastract}
\end{figure}

To screen for expression divergence of any of the \textit{S. cerevisiae} environment-specific regulators, we first asked whether any environment-specific regulators themselves showed expression divergence between species. In each environment, we identified each regulator as any gene known to regulate any number of downstream target genes in that environment. We found that many regulators, including many highly connected regulators, were diverging in plasticity (Fig \ref{fig:yeastract}B). Of the top 27 regulators (top 5 regulators from each environment, Cold Stress only has 2 total), 11 were identified as diverged in plasticity between species. For example, PHO4, the primary regulator of the Low Phosphate response, only shows expression plasticity in response to the Low Phosphate environment in \textit{S. cerevisiae} (Fig \ref{fig:yeastract}C).
We lastly asked whether the expression divergence observed in environment-specific regulators was coupled to expression divergence in their regulons (target genes). To screen for expression divergence in any of the genes downstream of these regulators, we repeated the above process instead identifying the regulons of each environment-specific regulator. We found that these regulons tended to have strongly conserved and plastic expression in the environment they were identified in, which we refer to as the “home” environment (Fig S\ref{fig:stackedbars}). In at least one other environment, however, which we refer to as the “alternative” environment, these regulons were seen to have remarkably divergent expression plasticity (Fig \ref{fig:yeastract}C-E). For example, the PHO4 regulon had strongly conserved and plastic expression in response to a Low Phosphate environment but divergent plasticity in the Diauxic Shift environment (Fig \ref{fig:yeastract}D). In all ``alternative” environments, the hybrid allelic expression matched one parental species’ expression, suggesting that one parent’s \textit{trans} regulatory environment has a dominant effect on regulon expression. Which parental species it was that had this dominant effect depended on the regulon.

\section{Conclusion}

Here we present a framework for comparing transcriptomic responses of different genotypes to a common environmental shift. We apply this framework to characterizing expression divergence between two species of yeast. We independently measure the two types of expression divergence, which we call level and plasticity, and find that each is associated with separate genetic mechanisms, \textit{cis} and \textit{trans} regulatory variation respectively. We find that level divergence is consistent between environments while plasticity divergence tends to be unique to a single environment. Finally, we see that environment-specific regulators often have diverged expression, but their regulons have conserved expression. Those same regulons, however, can have divergent plasticity in other environments where they have less-documented roles in responding to. Collectively our work gives a broad survey of the two modes of expression divergence and associates them with distinct molecular mechanisms and environmental behaviors.

\section{Data Availability}

\subsection{Transcriptomic data}
Raw RNA sequencing data from Krieger et al. 2020 and Lupo et al. 2021 are available at NCBI BioProject database (https://www.ncbi.nlm.nih.gov/bioproject/) under accession number PRJNA592756. Raw reads from Fay et al. 2023 are available under accession number PRJNA909640. All scripts used to align counts and analyze data are available at https://github.com/wyldtype/Redhuis2025. Only wildtype samples were used for this project.

\subsection{Yeastract regulatory matrices}
Binary regulatory matrices were downloaded from yeastract.com filtering by the following environmental conditions:

Control (un-stressed): Unstressed log-phase growth (control)

Diauxic Shift: Carbon source quality/availability $>$ non-fermentable carbon source (while there is a Diauxic Shift-specific environment, it only has 30 connections)

Hydroxyurea Shock: Stress $>$ Drug/chemical stress exposure

Low Nitrogen: Nitrogen source quality/availability $>$ Nitrogen starvation/limitation

Low Phosphate: Ion/metal/phosphate/sulfur/vitamin availability $>$ Phosphate limitation

Heat Stress: Stress $>$ Heat shock

Cold Stress: Stress $>$ Cold shock

\section{Methods}

\subsection{Mapping and quantifying transcriptome data}

Batch scripts used to align reads in this section are performed in the scripts 
$\texttt{cut\_map\_count\_tagseq.sbat}$ and $\texttt{cut\_map\_count\_fay.sbat}$, available in the github repository (https://github.com/wyldtype/Redhuis2025). STAR v2.7.11b was used for both aligning and quantification. Prior to aligning, STAR was run in --runMode genomeGenerate with the following parameters: --genomeSAindexNbases 11 --sjdbGTFfeatureExon gene --sjdbGTFtagExonParentGene Name (adding parameter --sjdbOverhang 74 for Fay et al. 2023) to generate reference genome indices.

Genome assemblies and gene annotations for \textit{S. cerevisiae} (S288C) and \textit{S. paradoxus} (CBS432) were downloaded from yjx1217.github.io/Yeast\_PacBio\_2016/data/ \cite{Yue2017}. Genome assemblies for \textit{S. kudriavzevii} (GCA\_947243775.1) and \textit{S. uvarum} (GCA\_027557585.1) were downloaded from NCBI. The RNAseq data from Fay et al. 2023 was pooled with samples including \textit{S. kudriavzevii} and \textit{S. uvarum} and therefore needed to be mapped to a concatenated genome that included all four species. Only reads for the \textit{S. cerevisiae} and \textit{S. paradoxus} alleles were quantified and included in this study.

50bp 3' tag-seq reads from Krieger et al. 2020 and Lupo et al. 2021 were mapped to the \textit{S. cerevisiae} and \textit{S. paradoxus} concatenated genome using STAR with the following parameters: --alignEndsType EndToEnd --outFilterScoreMinOverLread 0  --outFilterMatchNminOverLread 0 --outFilterMismatchNmax 4  --scoreGap -10. STAR was run in --quantMode GeneCounts to concurrently count reads within 500 base pairs of the transcription end site ($\pm$500 TES). An average of 60\%, or 1 million reads, were uniquely mapping per sample. Only uniquely mapping reads overlapping $\pm$500 of an annotated TES were quantified.

75bp RNAseq reads from Fay et al. 2023 were mapped to the \textit{S. cerevisiae}, \textit{S. paradoxus}, \textit{S. kudriavzevii} and \textit{S. uvarum} concatenated genome using STAR v2.7.11b with the following parameters: --alignEndsType EndToEnd --outFilterScoreMinOverLread 0  --outFilterMatchNminOverLread 0 --outFilterMismatchNmax 6  --scoreGap -10. STAR was run in --quantMode GeneCounts to concurrently count reads overlapping an annotated gene. While these reads were from paired-end data, they were mapped as single-end for consistency with the tagseq data. Counts for reads 1 and 2 were averaged after determining that read 1 and read 2 had highly correlated gene counts (average R=0.98, minimum R=0.9). An average of 84\%, or 10 million reads, were uniquely mapping per sample. Only uniquely mapping reads overlapping an annotated gene were quantified.

\subsection{Data cleaning}
All steps in this section are performed in the data cleaning script,  $\texttt{clean\_data.R}$ , available in the github repository. 

Prior to analysis, we assessed allele-specific mapping bias by measuring the percent of reads mapping to the wrong genome in the samples from a single parent. As the Fay et al. data had parental samples pooled together, this was only possible to assess with the Krieger et al. and Lupo et al. tagseq data. We identified 4/529 \textit{S. cerevisiae} samples and 5/576 \textit{S. paradoxus} samples with less than 90\% of reads mapping to the correct parental allele, average percent across all 4491 genes with a mean expression level greater than 30 counts per million. 6/9 of these samples had smaller than average library sizes, suggesting that filtering for small library sizes is an effective measure to mitigate sample-wide mapping bias. We also assessed mapping bias on a gene-by-gene basis across samples. Here we identified 36/4491 genes in \textit{S. cerevisiae}, 50/4491 genes in \textit{S. paradoxus}, and  3/4491 genes in both species (TDH1, ESC2, and YCL048W-A) with less than 90\% of reads mapping to the correct parental allele. Of note, we initially identified over 500 genes with mapping bias, due to soft-clipping in STAR aligning fragments of reads ($\leq$ 5 base pairs) to the wrong species' allele. We disabled soft-clipping with the parameter --alignEndsType EndToEnd. 

We removed samples with library sizes smaller than 100,000 total reads. Seven of these samples came from the Lupo et al. cell cycle environment (3 cerevisiae, 2 paradoxus, and 2 hybrid samples at various timepoints). We additionally removed 2 hybrid samples from the Lupo et al. cell cycle environment for having significantly different library sizes for the cerevisiae and paradoxus alleles (1,665,985 Scer reads vs 19,058 Spar reads and 425,558 Scer reads vs 190,710 Spar reads).

Count data was normalized to counts per million. The Krieger et al. 2020 and Lupo et al. 2021 data is from 3' tagmentation RNAseq, which is stranded and polyA-selected, where one read is equivalent to one transcript, so normalizing one gene's expression in one sample is as follows:

$$ gene_i \text{ (counts per million)} = \frac{\text{ reads mapped to } gene_i}{\text{library size}}*10^6 $$.

The Fay et al. 2023 RNAseq data is paired-end and also stranded and polyA-selected, but multiple reads may result from the same transcript. Because of this difference, to convert to counts per million, we additionally normalize by transcript length using the following equation from \cite{Zhao2020}:

$$ gene_i\text{ (counts per million)} = \frac{\frac{\text{reads mapped to }gene_i}{gene_i\text{ length}}}{\text{length-normalized library size}}*10^6 $$

where 

$$\text{length-normalized library size} = \sum_{j=1}^{nGenes} \frac{\text{reads mapped to }gene_j}{gene_j\text{ transcript length}}$$.


We lastly filtered out all genes that did not have a mean count of 30 counts per million in any experiment or allele, which brought us from a total of 5359 genes to 4997.

\subsection{Generalized linear model to determine divergence in level}

All steps in this section are performed in the $\texttt{single\_gene\_model\_construction\_and\_QC.R}$ script available in the github repository. For each gene $i$, in each experiment $j$, we fit the following negative binomial generalized linear model with a log link function using the glm.nb function from the R package MASS \cite{MASS}:
$$Y_{ij} = e^{tp*\beta_{timepoint_{ij}} + al*\beta_{allele_{ij}}} + \epsilon_{ij} $$

Where $Y_{ij}$ is the read count (in counts per million) of gene $i$ in expeirment $j$ at a given timepoint for a given allele (cerevisiae or paradoxus), $\epsilon_{ij}$ is the response residual (actual - predicted value of $Y_{ij}$), and $\beta_{timepoint_{ij}}$ and $\beta_{allele_{ij}}$ are the coefficients for the continuous variable timepoint ($tp$, ranging from 0 to around 1000 depending on the experiment, in minutes) and the categorical variable allele ($al$, 0 for paradoxus, 1 for cerevisiae).

From this model, we extracted each $\beta_{allele_{ij}}$, the estimated (natural) log-fold change associated with switching from the paradoxus to the cerevisiae allele at any given timepoint for gene $i$ in experiment $j$. We converted this to a log2 fold change, $\beta^*_{allele_{ij}}$, to be consistent with prior studies using the change of base formula: $$\beta^*_{allele_{ij}} = \frac{\beta_{allele_{ij}}}{log(2)}.$$

A gene $i$ in experiment $j$ was determined to be diverging in expression level if $|\beta^*_{allele_{ij}}| > 1$ and its associated Wald test p-value was less than $1*10^{-5}$ (Bonferroni correction for multiple testing on 4997 genes).

% TODO: continue writing analysis methods as you go through scripts to ready them for publication

\subsection{Correlation clustering to determine divergence in dynamcis}

All steps in this section are performed in the $\texttt{clustering.R}$ script available in the github repository. The input data for correlation clustering is a counts table where rows are genes, columns are samples, and all values are in counts per million. We ran separate clustering for each of the six experiments in the dataset (Cell Cycle, Saturated Growth, Low Nitrogen, Low Phosphorus, Heat Stress, and Cold Stress). To reduce expression noise prior to clustering, we took each gene's mean count across replicates at each timepoint. The Low Phosphorus and Saturated Growth experiments did not have replicates, so we instead took a moving average, making each timepoint the average of itself and its two neighboring upper and lower timepoints (a sliding window of 5).

For each of the six experiments, prior to clustering, we first identified two groups of genes: 1) lowly expressed genes whose average expression was less than 30 counts per million, and 2) lowly varying genes whose average expression was greater than 30 counts per million but whose dispersion (var/mean) was less than 1. We determined these two thresholds by plotting each gene's log2(var(expression)) against its log2(mean(expression)). We set thresholds on this plot to separate out as many genes as possible with abnormally high variance given their mean expression (lowly expressed) and genes with abnormally low variance given their mean (lowly varying). Note that in the data cleaning section we already pre-filtered lowly expressed genes that had expression less than 30 counts per million in both species in all six experiments, but now we are filtering additional genes that are lowly expressed in the focal experiment in one (or both) species. We labeled these lowly expressed and lowly varying genes and excluded them from clustering.

Using all remaining genes, we constructed a single correlation matrix in each of the six experiments. Each [i,j] entry in the correlation matrix is expression correlation of gene i and gene j in that experiment, where i and j can be orthologs of the same gene---one from cerevisiae and one from paradoxus. To assign these genes to clusters with correlated expression, we performed bottom-up hierarchical clustering on the correlation matrix (technically on $-1*\text{the correlation matrix}$, because hierarchical clustering attempts to minimize distance between objects in clusters). 

In bottom-up hierarchical clustering, each gene begins as its own cluster and at each step the pair with the strongest correlation are joined into the same cluster. After the first step, groups of genes that are joined together are summarised by the average of their pairwise correlations with a candidate next gene or gene group to determine whether they should be the next pair to be joined.

Once all genes that went into the clustering were joined together into one gene tree, we cut the tree to determine the number of clusters. Hierarchical clustering does not produce a specified number of clusters, unlike k-means. To determine the number of clusters, we began at the top branch, the final branch to be connected that joined all genes. For each of the 20 branches below this top branch, we calculated branch height as the correlation at which this branch was connected minus the correlation at which the previous, lower branch was connected. We then calculated the change in branch height as the height of each branch minus the height of the previous, lower branch. We chose to cut the branch with the maximum change in branch height. That is, the branch immediately above the branch that had the most significant drop off in variance explained, similar to the commonly used Gap Statistic \cite{Tibshirani2001}.

\clearpage
\section{Supplemental Figures}
\setcounter{figure}{0}

\begin{figure}[H]
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Supplement/QC_Clusters.png}}
    \caption{\textbf{Comparing results with 2, 3, or 4 clusters per environment.}}
    \label{fig:qc_clusters}
\end{figure}

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Supplement/AllEnvironmentsAvgExpr.png}}
    \label{fig:all_environments}
\end{figure}

\begin{figure}
    \centering
    \caption{\textbf{Level and plasticity divergence in all 6 environments. }In all 6 plots, the example conserved group. (A) are genes that increased in both species. The example diverged plasticity group (C) are genes that increased in \textit{S. paradoxus} and decreased in \textit{S. cerevisiae}. Expression in this group only is centered and scaled counts per million as opposed to log2(counts per million), to emphasize the difference in the plastic response between species. This is not a valid normalization scheme in the level-diverging groups, as centering will eliminate the difference in mean expression level between species. The example diverged level group (D) are genes that had higher expression level in \textit{S. cerevisiae}. The example diverged level and plasticity group (E) are genes that are higher expressed in \textit{S. paradoxus} and switched from the static (lowly-varying) cluster in \textit{S. cerevisiae} to the increasing cluster in \textit{S. paradoxus}. Genes are ordered by the 4 divergence categories separately in each of the six environments.}
    \label{fig:all_environments}
\end{figure}

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Supplement/LevelvsPlasticityBoxplots.png}}
    \caption{\textbf{Genes diverging in expression plasticity do not have significantly different magnitudes of log2 fold change versus genes with conserved plasticity.} In each plot, genes are grouped by their cluster in each species (increasing=1, decreasing=2, or static=0) and by whether their log2 fold change was significantly higher in S.cer (1) or higher in S.par (-1). 20Only two t.tests were performed per environment, one for higher expressed in S.cer and one for higher expressed in S.par.}
    \label{fig:boxplots}
\end{figure}

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Supplement/LevelThresholdBars.png}}
    \caption{\textbf{Proportion of genes with divergence in expression level is highly dependent on chosen fold change cutoff.} In each of the 6 environments, the proportions of genes belonging to each class of expression divergence are shown compared for three different thresholds on fold change magnitude. Top row: any gene with a DESeq2 adjusted p-value < 0.05 is considered to be diverging in expression level. Middle row: genes with a DESeq2 adjusted p-value < 0.05 and estimated mean expression (fold change) that is at least 1.5x higher in one species than the other are considered to be diverging in expression level. Bottom row: genes with a DESeq2 adjusted p-value < 0.05 and estimated mean expression (fold change) that is at least 2x higher in one species than the other are considered to be diverging in expression level.}
    \label{fig:threshold}
\end{figure}

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Supplement/tp0.png}}
    \caption{\textbf{Genes with expression plasticity in opposite directions (increasing or decreasing) tend to have differences in starting mRNA abundance in all 6 environments.} A) (left) The plastic expression response of hypothetical gene A moves in opposite directions in each species in response to the “home” environmental shift. Red bar indicates difference in mRNA abundance at timepoint 0 (standard laboratory conditions). (right) In the five other environments, it is not guaranteed that the same difference in starting mRNA abundance will be present. B) Observed differences in starting mRNA abundance for each gene that decreased in one species and increased in the other. Red line indicates observed value, grey distributions represent 10,000 random samplings (with replacement) of the same number of genes as observed.}
    \label{fig:tp0}
\end{figure}

\begin{figure}
    \centering
    \fbox{\includegraphics[width=\textwidth]{chapters/figures/Plasticity/Supplement/YeastractStackedBars.png}}
    \caption{\textbf{Genes with conserved expression plasticity are more likely to be targets of regulation.} A) Each set of three stacked bars represents i) the proportions of genes belonging to each plasticity group, ii) the proportions of genes that are targets of regulation in the unstressed regulatory network, iii) the proportions of genes that are targets of regulation in the portion of the regulatory network specific to each environment. B) Same as A except counting the source of each regulatory interaction (regulator) rather than the target.}
    \label{fig:stackedbars}
\end{figure}


